### PR TITLE
Release MKE 4.0.0-alpha.2

### PR DESCRIPTION
**Pull request by byDimasik**

### CODE REVIEW

File: release-matrix/release-matrix.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-    "mkeOperatorVersion": "v1.2.0",
+    "mkeOperatorVersion": "v1.2.1",
     "k0sVersion": "v1.31.1+k0s.1",
     "isLatest": true,
+    "minimumMkectlVersion": "v4.0.1-alpha.2"
+  },
+  {
+    "mkeReleaseVersion": "v4.0.1-alpha.1",
+    "blueprintOperatorVersion": "v1.0.17",
+    "mkeOperatorVersion": "v1.2.0",
```
Comment (byDimasik) [inline release-matrix/release-matrix.json:13]: @RaunakShah 
how is this supposed to work if all versions always pull the latest MOP? Should I also set it to v1.2.1 even though when the first alpha was released, the latest MOP version was v1.2.0?

File: release-matrix/release-matrix.json [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    "mkeReleaseVersion": "v4.0.1-alpha.2",
     "blueprintOperatorVersion": "v1.0.18",
-    "mkeOperatorVersion": "v1.2.0",
+    "mkeOperatorVersion": "v1.2.1",
     "k0sVersion": "v1.31.1+k0s.1",
     "isLatest": true,
+    "minimumMkectlVersion": "v4.0.1-alpha.2"
+  },
+  {
+    "mkeReleaseVersion": "v4.0.1-alpha.1",
```
Comment (tppolkow) [inline release-matrix/release-matrix.json:11]: should we still have this one on the matrix, even though its broken?

File: release-matrix/release-matrix.json (hunk 1 above)
Comment (RaunakShah) [reply release-matrix/release-matrix.json:13]: No it should remain as `v1.2.0`. All versions pull the latest at the time of their release. 
So if a user wants to deploy `v4.0.1-alpha.1`, they should always get MOP `v1.2.0`.

File: release-matrix/release-matrix.json (hunk 2 above)
Comment (RaunakShah) [reply release-matrix/release-matrix.json:11]: I don't mind having it removed since we don't want anyone deploying it.

Comment (byDimasik) [reply release-matrix/release-matrix.json]: removed

**Review by tppolkow — APPROVED** [review]

**Pull request merged**